README.md

```diff
@@ -32,38 +32,18 @@ Clone the respository and ensure you have the requirements below.
 
 ## Configuration Setup
 
-1. Copy the values from the `firebaseConfig` variable in the web app in the firebase console to the `FirebaseConfigs` variable in `./config/app.js`.
-2. Download the `GoogleService-Info.plist` file from the ios app on firebase. Place this file in `./mobile/configs/app`.
-3. Copy the `GOOGLE_APP_ID` and `API_KEY` from the plist file you just downloaded into the ios object in the `FirebasePlatformOverrides` variable in `./config/app.js`. Then copy the `CLIENT_ID` from the same plist file to line 48 in the `GoogleConfigs` variable in `./config/app.js`.
-4. Copy the `REVERSED_CLIENT_ID` from the same plist file into line 39 in `./mobile/app.json`.
-5. Download the `google-services.json` file from the android app on firebase. Place this file in `./mobile/configs/app` and in `./mobile/android/app`.
-6. Copy the `current_key` and `mobilesdk_app_id` from the json file you just downloaded into the android object in the `FirebasePlatformOverrides` variable in `./config/app.js`. Then copy the `client_id` on line 18 to line 47 of the same `app.js` file.
-7. Again from the downloaded json file, copy the `current_key` and the `certificate_hash` to lines 54 and 55 of `./mobile/app.json`.
-8. Create `.runtimeconfig.json` in `./server/functions` with the following:
-
-	```
-	{
-	  "envs": {
-	    "project_name": "[appName]",
-	    "sendgrid_api_key": "...",
-	    "sendgrid_emails_validation_api_key": "...",
-	    "is_prod": "false", // set false for staging,
-	    "devlogin": true,
-	    "twilio_phone_number_from": "...",
-	    "twilio_auth_token": "...",
-	    "twilio_account_sid": "..."
-	  }
-	}
-	```
-9. Add the Sendgrid API key to the `.runtimeconfig.json` file you just created (get this key from a dev on the team).
-10. Now create `.env` in `./server/functions` with the following:
+1. Download the `GoogleService-Info.plist` file from the ios app on firebase. Place this file in `./mobile/configs/app`.
+2. Download the `google-services.json` file from the android app on firebase. Place this file in `./mobile/configs/app` and in `./mobile/android/app`.
+3. Copy the `.env-starter` file to `.env` in the root directory, and fill in each variable with the correct values. Alternatively, obtain an environment file from another developer on the team.
+4. From the root directory, run `yarn env:set`. This will generate all of the configuration files
```

Comment:
When I ran env:set, the envsub command used in the script couldn’t be found because I didn’t have the package installed (it's installed in the setup script). We should add instructions to install this package before running env:set. (npm install envsub)

Reply:
This worked ok for me, I didn't have to run `nom install envsub`. Isn't the expectation to run the setup script before calling env:set?

Reply:
I tried an instance where I ran the setup script first and I got this error, because env:set sets up the project.pbxproj file I believe. Did you not get this?

![image](https://user-images.githubusercontent.com/47067364/107135245-db21fa00-68ad-11eb-9299-26bc64949512.png)

Reply:
Oh yes I did. I just thought it was because the initial ReadMe missed a step. But I see the readme still has the `env:set` before the setup script. My bad

Reply:
@cara-wong @crugwiro This occurred to me shortly after finishing the instructions and I was hoping you guys would expose the problem. I'll simply add a step to run `yarn` to install dependencies before everything.

Reply:
hey @rlnsy I tried an instance where I ran "yarn all" first before the env:set command, but that also errors out because it requires the persona and react-native-switch dependencies to be cloned first. (cloned in the setup script)